hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -149,6 +160,23 @@ private void cleanupCache(long lastRatisTransactionIndex) {
     omMetadataManager.getBucketTable().cleanupCache(lastRatisTransactionIndex);
   }
 
+  /**
+   * Set OzoneManagerDoubleBuffer metrics values.
+   * @param flushedTransactionsSize
+   */
+  private void setOzoneManagerDoubleBufferMetrics(
+      long flushedTransactionsSize) {
```

Comment:
NIT: can we rename this method to something like updateMetrics?

Reply:
Done.